README.rst

```diff
@@ -19,28 +19,43 @@ OpenWISP 2 monitoring module (Work in progress).
 
 ------------
 
-Install Depdendencies
+Install Dependencies
 ---------------------
 
-`Install influxdb <https://docs.influxdata.com/influxdb/v1.4/introduction/installation/>`_ eg:
+We use InfluxDB to store metrics and Redis as celery broker (you can use a different
+broker if you want). The recommended way for development is running them using Docker
+so you will need to `install docker and docker-compose <https://docs.docker.com/engine/install/ubuntu/>`_
```

Comment:
We should provide generic installation link rather than a particular distribution. https://docs.docker.com/engine/install/